ios/use_react_native-0.62.rb

```diff
@@ -81,6 +81,18 @@ def include_react_native!(react_native:, target_platform:, project_root:, flippe
   try_pod('boost-for-react-native',
           "#{react_native}/third-party-podspecs/boost-for-react-native.podspec",
           project_root)
+
+  return unless target_platform == :macos
+
+  # Hermes
+  begin
+    hermes_engine_darwin = resolve_module('hermes-engine-darwin')
+    pod 'React-Core/Hermes', :path => "#{react_native}/"
+    pod 'hermes', :path => Pathname.new(hermes_engine_darwin).relative_path_from(project_root).to_s
```

Comment:
I assume this is the one you’re potentially expecting to throw? Perhaps make it the first line of this block so the above pod doesn’t still get activated.

Reply:
`resolve_module` is the one throwing if it cannot find `hermes-engine-darwin`.

Reply:
Ah, perfect 👍